tl-tl/node/api.md

```diff
@@ -1,10 +1,10 @@
-# API Reference
+# Reperensiya ng API
 
-Each node in the Neo-CLI provides an API interface for obtaining blockchain data from a node, making it easy to develop blockchain applications. The interface is provided via [JSON-RPC](http://wiki.geekdream.com/Specification/json-rpc_2.0.html), and the underlying protocol uses HTTP/HTTPS for communication. To start a node that provides an RPC service, run the following command:
+Bawat node sa Neo-CLI ay nagbibigay ng isang interface ng API para makamit ang datos ng blockchain na mula sa isang node, na pinapadali nito na makabuo ng mga blockchain application. Ang interface ay ibinigay sa pamamagitan ng [JSON-RPC](http://wiki.geekdream.com/Specification/json-rpc_2.0.html), at ang pinagtitibayan na protocol ay gumagamit ng HTTP/HTTPS para sa komunikasyon. Para makapagsimula ng isang node na nagbibigay ng isang RPC na serbisyo, i-run ang mga sumusunod na command:
 
 `dotnet neo-cli.dll /rpc`
 
-To access the RPC server via HTTPS, you need to modify the configuration file config.json before starting the node and set the domain name, certificate, and password:
+Para ma-akses ang serber ng RPC via HTTPS, kailangan mong baguhin ang kompigurasyon na payl config.json bago simulan ang node at i-set ang pangalan ng domain, sertipikato, at password:
```

Comment:
Para (makapasok) sa server ng RPC.  

sertipikato, at kontrasenyas

Reply:
leave password as is.. don't translate it
it is sertipiko hindi sertipikato